docs/acme-divergences.md

```diff
@@ -4,61 +4,61 @@ While Boulder attempts to implement the ACME specification as strictly as possib
+## [Section 6.1.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1.2)
 
-Boulder does not implement the `status`, `applications`, or `certificates` fields
+Boulder does not implement the `terms-of-service-agreed`, `applications`, or `certificates` fields
 in the registration object (nor the endpoints the latter two link to).
 
-## [Section 6.1.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.3)
+## [Section 6.1.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1.3)
 
 Boulder does not implement applications, instead it implements the `new-cert` flow from [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5). Instead of application requirements Boulder currently uses authorizations that are created using the `new-authz` flow from [draft-ietf-acme-02 Section 6.4](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.4).
 
-## [Section 6.1.4.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.4)
+## [Section 6.1.4.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1.4)
 
 Boulder does not implement the `scope` field in authorization objects.
 
-## [Section 6.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2)
+## [Section 6.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.3)
 
-Boulder does not allow `tel` URIs in the registrations `contact` list.
+Boulder only allows `mailto` URIs in the registrations `contact` list.
 
-## [Section 6.2.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2.1)
+## [Section 6.2.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.3.1)
 
-Boulder implements draft-03 style key roll-over with a few divergences. Since Boulder doesn't currently use the registration URL to identify users we do not check for that field in the JWS protected headers but do check for it in the inner payload. Boulder also requires the outer JWS payload contains the `"resource": "key-change"` field.
+Boulder implements draft-04 style key roll-over with a few divergences. Since Boulder doesn't currently use the registration URL to identify users we do not check for that field in the JWS protected headers but do check for it in the inner payload. Boulder also requires the outer JWS payload contains the `"resource": "key-change"` field.
 
-## [Section 6.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.3)
+## [Section 6.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.4)
 
 Boulder does not implement applications, instead it implements the `new-cert` flow from [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5). Instead of application requirements Boulder currently uses authorizations that are created using the `new-authz` flow from [draft-ietf-acme-02 Section 6.4](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.4).
```

Comment:
With draft-04 we also need to describe our divergence from the proactive issuance portion:
> The server MUST issue the requested certificate and update the
> application resource with a URL for the certificate as soon as the
> client has fulfilled the server's requirements.

Let's Encrypt clients need to continue explicitly asking for the certificate to be issued using `new-cert`.